full.async/src/full/async.clj

```diff
@@ -62,6 +68,26 @@
              (recur (dec retries#)))
            res#)))))
 
+(defmacro go-try> [err-chan & body]
```

Comment:
What we did with go-try and <? was to introduce the minimum viable approach to error handling. Since anything in Java can throw the way core.async deals with exceptions isn't really sufficient for production-level code.

When we use go-try and <? everywhere, the exception normally gets popped to appropriate level in the system where it gets handled appropriately.

This is of course really just mimics Java's general approach to error handling.